moped-editor/src/queries/dashboard.js

```diff
@@ -0,0 +1,44 @@
+import { gql } from "@apollo/client";
+
+export const USER_FOLLOWED_PROJECTS_QUERY = gql`
+  query UserFollowedProjectsQuery($userId: Int) {
+    moped_user_followed_projects(where: { user_id: { _eq: $userId }, project: { is_retired : { _neq: true } } }) {
+      project {
+        project_id
+        project_name
+        current_phase
+        current_status
+        moped_proj_notes(
+          where: { project_note_type: { _eq: 2 } } 
+          order_by: { date_created: desc }
+        ) {
+          added_by
+          project_note_type
+          status_id
+          project_note
+        }
+      }
+    }
+  }
+`;
+
+export const USER_PERSONNEL_PROJECTS_QUERY = gql`
```

Comment:
should this be USER_PERSONAL_PROJECTS_QUERY?

Reply:
I named it that because it queries the `moped_proj_personnel` table as it relates to `moped_project` but I'm open to renaming it